Controller/ElFinderController.php

```diff
@@ -184,8 +185,12 @@ public function loadAction(Request $request, $instance, $homeFolder)
     {
         $httpKernel = $this->get('http_kernel');
 
-        $preExecutionEvent = new ElFinderPreExecutionEvent($request, $httpKernel, $instance, $homeFolder);
-        $this->get('event_dispatcher')->dispatch(ElFinderEvents::PRE_EXECUTION, $preExecutionEvent);
+        try {
+            $preExecutionEvent = new ElFinderPreExecutionEvent($request, $httpKernel, $instance, $homeFolder);
```

Comment:
I agree with @cordoval , I think its better to throw the exception without cathing it here. Its up to your app to catch it and to deal with it.